### PR TITLE
added robotino models with softhand

### PR DESCRIPTION
**Pull request by ghost**
As mentioned in the previous pull request, those are the new models with the softhand.

### CODE REVIEW

File: robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  </node>
+
+  <group ns="real/robotino">
+	  <node name="robot_controller_node" pkg="uibk_robot_driver" type="robot_controller_node" output="screen">
+	  	  <remap from="cmd_rotatory" to="/cmd_rotatory"/>
+	  </node>
+    <node name="joint_states_relay" pkg="topic_tools" type="relay" args="joint_control/get_state /joint_states" cwd="node" respawn="true" output="screen"/>
+  </group>
+
+  <group ns="hand_controller">
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch]: you dont need to launch the kcl hand control

File: robotino_bringup/robots/uibk-robotino2-sh/urdf/robot.gv [hunk 2]
```diff
@@ -0,0 +1,182 @@
+digraph G {
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2-sh/urdf/robot.gv]: do you need this file? if not , please remove

File: robotino_bringup_sim/robots/uibk-robotino2-sh/uibk-robotino2-sh.launch [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <param name="tf_prefix" type="string" value="" />
+  </node>
+
+  <include file="$(find robotino_controller_configuration_gazebo)/launch/joint_state_controller.launch" />
+
+  <include file="$(find robotino_controller_configuration_gazebo)/launch/base_controller.launch" />
+
+  <include file="$(find robotino_controller_configuration_gazebo)/launch/arm_controller.launch" />
+
+  <include file="$(find robotino_controller_configuration_gazebo)/launch/hand_controller.launch" />
```
Comment (NadiaHG) [inline robotino_bringup_sim/robots/uibk-robotino2-sh/uibk-robotino2-sh.launch]: this hand control is for the kclhand, please remove it

**Comment by ipa-nhg** [comment]
@lokalmatador actually the new robot uibk-robotino2-sh is  uibk-robotino2 but with a different hand,  you need only to modify the robot description and the launch file, I would prefer use softlinks from squirrel_robotino/robotino_bringup/robots/uibk-robotino2 to squirrel_robotino/robotino_bringup/robots/uibk-robotino2-sh for all the other files, because in case that you calibrate the camera or change the controller of the base (common components) , the changes will be automatically done for both robots. what do you think? If you are agree I can create a new pull request based on yours.

**Comment by ghost** [comment]
@ipa-nhg I did the changes to the package.xml and CMakelists.txt and also the launch files. What you are suggesting sounds good, got for it! Thanks for the help!

**Review by ipa-nhg — APPROVED** [review]

**Comment by ipa-nhg** [comment]
Added an Issue https://github.com/squirrel-project/squirrel_robotino/issues/155

**Pull request merged**